src/main/java/io/fabric8/jenkins/openshiftsync/GlobalPluginConfiguration.java

```diff
@@ -114,6 +116,28 @@ private void configChange() {
         OpenShiftUtils.initializeOpenShiftClient(server);
         this.namespace = getNamespaceOrUseDefault(namespace, getOpenShiftClient());
 
+        logger.info("Waiting for Jenkins to be started");
```

Comment:
I am wondering why you need to do this here? Can you not wait until all jobs are loaded? Take a look at this: http://javadoc.jenkins.io/hudson/model/listeners/ItemListener.html#onLoaded()